### ISSUE TITLE
Document ID generation

### ISSUE THREAD

**Issue by sigmapointindia** [freelancer4you/godigital#4]
No files created are mapped with serial numbers.
Serial Nos. can be allotted to the file(s) and/or the filesystem may choose document numbers based on a series of 10 digits.

**Comment by freelancer4you** [comment]
So this means that every file should receive a serial number and the format of the serial number should be given by the user or the administrator?

**Comment by sigmapointindia** [comment]
Yes, the serial number for document is a must. The format can be given by Administrator/User or if it is not possible for you to work on that now, make sure every document generated at each user has a serial number based on levels.
For example, A user of department 1 created file can have serial number allotted like 9100001001
The document if successfully approved will be having serial number at Level 2 as 9100001011, in case of rejection 9100001021. And at the time of approval at level 3, 9100001111
And the next document generated will have 9100002001, at Level 2 accepted - 9100002011; Level 3 accepted 9100002111 or rejected - 9100002211
The first digit 9 is for a "department", next 6 numbers for the series of document and last 3 numbers for approved or reject status. File Viewer Table should show all the files as it is created or waiting for approval, or approved or certified. unlike now, it should stay in the view field and one can use filters to see the documents.
